EIPS/eip-4902.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: 4902
+title: Decentralized Autonomous Access (DAA)
+description: DAAs represent an access over digital or physical things & resources, with the ability to connect with entities that seeking to use it
+author: <Daniel @eartho-group>
+discussions-to: https://github.com/ethereum/eips/issues/4902
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-22
+requires: 20
+---
+
+## Abstract
+As humans, we are constantly seeking as much access to resources as possible, We also make access trades for our needs regularly, it can be for information, food, rides, content, hotels, jobs, and so on. The trade could be as simple as me looking for access to food and the other side looking for access to money.
```

Comment:
Several grammar errors here. Also, please do not use first or second person pronouns in EIP text.

Reply:
im not a native speaker, i used serval tools but its probably still not perfect. maybe you can give it your touch